src/main/java/uk/gov/hmcts/reform/cwrdapi/service/CwrCommonRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.cwrdapi.service;
+
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+@Repository
+public class CwrCommonRepository implements ICwrCommonRepository {
+
```

Comment:
rename of Cwr**d**Commonreporsitory

Reply:
Hello Shashank, let me rename it in next iteration. I have some other changes as well.

Reply:
The Class and interface is renamed.